packages/esm-outpatient-app/src/visit-header/visit-header.component.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { Button, Header, HeaderGlobalAction, HeaderGlobalBar, HeaderName } from 'carbon-components-react';
+import Search20 from '@carbon/icons-react/lib/search/20';
+import CloseFilled20 from '@carbon/icons-react/es//close--filled/20';
+import { useTranslation } from 'react-i18next';
+import styles from './visit-header.scss';
+import { age, usePatient } from '@openmrs/esm-framework';
+import capitalize from 'lodash-es/capitalize';
+
+interface VisitHeadeProps {}
+
+const VisitHeader: React.FC<VisitHeadeProps> = () => {
```

Comment:
This is drawing over the other headers so they are not available until you close the header, but why would we close the header?